### PR TITLE
DEP-3637 Close CJRS/SEISS DEP pages

### PR DESCRIPTION
**Pull request by AdamSmithHMRC**
# DEP-(3637)

## Checklist PR Raiser
 - [x]  I've ensured code coverage has not decreased
 - [x]  I've dealt with any new compilation warnings
 - [x]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [ ]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [x]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [x]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [x]  I've checked to ensure all relevant unit tests have been written
 - [x]  I've checked to ensure the team's coding standards have been met
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  If I merge I will ensure I use squash and merge

### CODE REVIEW

**Comment by AdamSmithHMRC** [comment]
Waiting for confirmation of removal of DA links from the COVID from [SASS-2953](https://jira.tools.tax.service.gov.uk/browse/DEP-2953)

File: app/views/webchat/JobRetentionSchemeView.scala.html [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-                @messages("job.retention.scheme.li2")
-            </a>
-        </li>
-    </ul>
-
-    @components.webchat_container(appConfig, webChatClient)
-    @components.webchat_loading_text()
-
-    <p class="margin-top">
-        <a class="govuk-link" id="return-link" href="@appConfig.jobRetentionSchemeReturnUrl">
```
Comment (tonymyers) [inline app/views/webchat/JobRetentionSchemeView.scala.html:60]: I don't think the jobRetentionSchemeReturnUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required

File: app/views/webchat/JobRetentionSchemeView.scala.html [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-
-    <h2 class="margin-top">@messages("global.webchat.title.h2")</h2>
-
-    <p>@messages("job.retention.scheme.p1")</p>
-    <p>@messages("job.retention.scheme.p2")</p>
-    <p>@messages("job.retention.scheme.p3")</p>
-
-    <ul class="bullets">
-        <li>
-            <a class="govuk-link" id="report-fraud-link" href="@appConfig.reportFraudulentActivityUrl">
```
Comment (tonymyers) [inline app/views/webchat/JobRetentionSchemeView.scala.html:45]: I don't think the checkCoveredByCoronavirusJRSUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required

File: app/views/webchat/JobRetentionSchemeView.scala.html [hunk 3]
```diff
@@ -1,65 +0,0 @@
...
-                @messages("job.retention.scheme.li1")
-            </a>
-        </li>
-        <li>
-            <a class="govuk-link" id="check-jrs-scheme-link" href="@appConfig.checkCoveredByCoronavirusJRSUrl">
```
Comment (tonymyers) [inline app/views/webchat/JobRetentionSchemeView.scala.html:50]: I don't think the checkCoveredByCoronavirusJRSUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required

File: app/views/va/VASupportForCoronavirusView.scala.html [hunk 4]
```diff
@@ -1,46 +0,0 @@
...
-    <h1 class="header-margin-top">@messages("va.global.h1.title")</h1>
-
-    <div class="margin-bottom">@components.webchat_container(appConfig, webChatClient)</div>
-    @components.webchat_loading_text()
-
-    <p class="margin-top">@messages("va.global.h1.bullet.title")</p>
-
-    <ul class="list list-bullet">
-        <li>
-            <a class="govuk-link" id="coronavirus-general-info-link" href="@appConfig.coronavirusGeneralInfoUrl">@messages("support.for.coronavirus.linkto.general.info")</a>
```
Comment (tonymyers) [inline app/views/va/VASupportForCoronavirusView.scala.html:39]: I don't think the coronavirusGeneralInfoUrl in app.config is used anywhere else. Can you please check and remove  from appconfig if required

File: app/views/va/VASelfEmploymentIncomeSupportSchemeView.scala.html [hunk 5]
```diff
@@ -1,38 +0,0 @@
...
-
-@main_template(title = messages("va.global.h1.title"), navTitle = Some(messages("global.nav.va.title"))) {
-
-    <h1>@messages("va.global.h1.title")</h1>
-
-    <div class="margin-bottom">@components.webchat_container(appConfig, webChatClient)</div>
-    @components.webchat_loading_text()
-
-    <div>
-        <a class="govuk-link" id="self-employment-income-support-scheme-link" href="@appConfig.selfEmploymentIncomeSupportReturnUrl">@messages("self-employment-income-support.li1")</a>
```
Comment (tonymyers) [inline app/views/va/VASelfEmploymentIncomeSupportSchemeView.scala.html:36]: I don't think the selfEmploymentIncomeSupportReturnUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required

File: app/views/webchat/SelfEmploymentIncomeSupportView.scala.html [hunk 6]
```diff
@@ -1,48 +0,0 @@
...
-        <p>@messages("global.opening.times.weekend.p")</p>
-    </div>
-
-    <h2 class="margin-top">@messages("global.webchat.title.h2")</h2>
-
-    @components.webchat_container(appConfig, webChatClient)
-    @components.webchat_loading_text()
-
-    <p class="margin-top">
-        <a class="govuk-link" id="return-link" href=@appConfig.selfEmploymentIncomeSupportReturnUrl>
```
Comment (tonymyers) [inline app/views/webchat/SelfEmploymentIncomeSupportView.scala.html:43]: I don't think the selfEmploymentIncomeSupportReturnUrl in app.config is used anywhere else. Can you please check and remove from appconfig if required

File: app/views/va/VASupportForCoronavirusView.scala.html [hunk 7]
```diff
@@ -1,46 +0,0 @@
...
-@this(
-    main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("va.global.h1.title"), navTitle = Some(messages("global.nav.va.title"))) {
-
-    <h1 class="header-margin-top">@messages("va.global.h1.title")</h1>
```
Comment (tonymyers) [inline app/views/va/VASupportForCoronavirusView.scala.html:30]: You need to remove any messages from the message file that are used on this page but not used anywhere else

File: app/views/va/VASelfEmploymentIncomeSupportSchemeView.scala.html [hunk 8]
```diff
@@ -1,38 +0,0 @@
...
-@import views.html.templates.MainTemplate
-
-@this(
-    main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("va.global.h1.title"), navTitle = Some(messages("global.nav.va.title"))) {
```
Comment (tonymyers) [inline app/views/va/VASelfEmploymentIncomeSupportSchemeView.scala.html:28]: You need to remove any messages from the message file that are used on this page but not used anywhere else

File: app/views/CUIViews/JobRetentionSchemeHelpView.scala.html [hunk 9]
```diff
@@ -1,40 +0,0 @@
...
-
-@import config.AppConfig
-@import templates.MainTemplateCUI
-
-@this(main_template: MainTemplateCUI, appConfig: AppConfig)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(
-    title = messages("cui.job.retention.scheme.title"),
```
Comment (tonymyers) [inline app/views/CUIViews/JobRetentionSchemeHelpView.scala.html:25]: You need to remove any messages from the message file that are used on this page but not used anywhere else

File: app/views/webchat/JobRetentionSchemeView.scala.html [hunk 10]
```diff
@@ -1,65 +0,0 @@
...
-@import uk.gov.hmrc.webchat.client.WebChatClient
-@import views.html.templates.MainTemplate
-
-@this(main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("job.retention.scheme.title")) {
```
Comment (tonymyers) [inline app/views/webchat/JobRetentionSchemeView.scala.html:27]: You need to remove any messages from the message file that are used on this page but not used anywhere else

File: app/views/webchat/JobRetentionSchemeView.scala.html [hunk 11]
```diff
@@ -1,65 +0,0 @@
...
-
-    <h1 class="header-margin-top">@messages("job.retention.scheme.title")</h1>
```
Comment (tonymyers) [inline app/views/webchat/JobRetentionSchemeView.scala.html:29]: You need to remove any messages from the message file that are used on this page but not used anywhere else

File: app/views/webchat/SelfEmploymentIncomeSupportView.scala.html [hunk 12]
```diff
@@ -1,48 +0,0 @@
...
-@import uk.gov.hmrc.webchat.client.WebChatClient
-@import views.html.templates.MainTemplate
-
-@this(main_template: MainTemplate,
-    appConfig: AppConfig,
-    webChatClient: WebChatClient)
-
-@()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
-@main_template(title = messages("self.employment.income.support.scheme.title")) {
```
Comment (tonymyers) [inline app/views/webchat/SelfEmploymentIncomeSupportView.scala.html:27]: You need to remove any messages from the message file that are used on this page but not used anywhere else

**Review by tonymyers — CHANGES_REQUESTED** [review]

**Review by tonymyers — APPROVED** [review]

**Review by tonymyers — APPROVED** [review]

**Review by tonymyers — APPROVED** [review]

**Pull request merged**